dali/kernels/signal/dct/dct_gpu.cu

```diff
@@ -29,11 +29,49 @@ namespace kernels {
 namespace signal {
 namespace dct {
 
+struct Lifter {
+  DALI_HOST_DEV
+  explicit Lifter(float coeff): coeff_(coeff) {}
+
+  DALI_HOST_DEV
+  float operator()(float val) { return val * coeff_; }
+
+  const float coeff_;
+};
+
+struct IdLifter {
+  DALI_HOST_DEV
+  float operator()(float val) { return val; }
+};
+
+template <bool nonzero>
+struct LiftersTable {};
+
+template <>
+struct LiftersTable<true> {
```

Comment:
```suggestion
struct LifterTable<true> {
```

Reply:
I think this class is not necessary at all. Why do we need this abstraction over a simple pointer to an array of coefficients?

Reply:
@mzient It's static optimization - to get rid of `if` in case of no liftering. But as Joaquin suggested I can just use `if` on the static parameter without all those classes

Reply:
I've remove LifterTabble